configure.ac

```diff
@@ -1150,6 +1150,14 @@ AC_ARG_WITH([pkgconfigdir],
     [pkgconfigdir='${libdir}/pkgconfig'])
 AC_SUBST([pkgconfigdir])
 
+AC_ARG_ENABLE([no-export],
+    [AS_HELP_STRING([--enable-no-export],
+        [Build libzmq with empty ZMQ_EXPORT macro, bypassing platform-based automated detection [default=no]])])
+if test "x$enable_no_export" = "xyes"; then
+    CPPFLAGS="-DZMQ_NO_EXPORT $CPPFLAGS"
+    CXXFLAGS="-DZMQ_NO_EXPORT $CXXFLAGS"
```

Comment:
CPPFLAGS is enough, CXXFLAGS are compiler flags for g++, but this is just preprocessor stuff

Reply:
Thanks, I wasn't sure.